rust/repositories.bzl

```diff
@@ -955,8 +958,12 @@ def _get_toolchain_repositories(name, exec_triple, extra_target_triples, version
 
         # Define toolchains for each requested version
         for channel in channels.values():
+            # Check if this toolchain is requested to be aliased.
+            full_name = "{}__{}__{}".format(name, target_triple, channel.name)
+            if full_name in aliases:
```

Comment:
Could you add a warning here for any aliases that go unused?

Reply:
Yeah it should just fail.  Added a check.